### PR TITLE
hack-vancouver23: Add landing page

### PR DESCRIPTION
**Pull request by razvand**
Add landing page for Vancouver Unikraft Hackathon, on September 23-24, 2023

### CODE REVIEW

File: content/en/community/hackathons/2023-09-vancouver/_index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [Shayan Hosseini](https://shayanh.com/)
+* [Alexandra (Sasha) Fedorova](https://people.ece.ubc.ca/sasha/)
+
+As part of the Unikraft community, [Răzvan Deaconescu](https://github.com/razvand/) will be on-site, with other community members providing support online, on Discord.
+
+### Schedule
+
+#### Saturday, September 23, 2023
+
+| Time (CET)    | Session                                         |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-09-vancouver/_index.md:41]: Are the times CET or Vancouver local time?

File: content/en/community/hackathons/2023-09-vancouver/_index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| 12:45 - 14:00 | Lunch                                           |
+| 14:00 - 15:00 | Running (Complex) Applications in Unikraft      |
+| 15:00 - 16:00 | Porting Applications to Unikraft                |
+| 16:00 - 16:15 | Break                                           |
+| 16:00 - 17:00 | Porting Applications to Unikraft                |
+| 17:00 - 17:45 | Guest Talk                                      |
+
+#### Sunday, September 24, 2023
+
+| Time (CET)    | Session                                             |
```
Comment (StefanJum) [inline content/en/community/hackathons/2023-09-vancouver/_index.md:56]: Are the times CET or Vancouver local time?

**Review by StefanJum — CHANGES_REQUESTED** [review]
See the comments. It's all good besides that.

**Comment by razvand** [comment]
@StefanJum , please see the updated version. Then we can merge it and start advertising.

**Review by StefanJum — COMMENTED** [review]
@razvand the links in the [registration form](https://forms.gle/mDumYG8pQtxcX3ES9) are marked as `TODO`s. We can publish and do the announcements after those are fixed.

**Comment by razvand** [comment]
> @razvand the links in the [registration form](https://forms.gle/mDumYG8pQtxcX3ES9) are marked as `TODO`s. We can publish and do the announcements after those are fixed.

@StefanJum , fixed. Take another look please.

**Review by StefanJum — APPROVED** [review]
All good.
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**